src/main/res/values/strings.xml

```diff
@@ -183,14 +183,14 @@
     <string name="new_applications">New apps</string>
     <string name="install_after_sync">Auto-install</string>
     <string name="install_after_sync_summary">Installs new versions of installed new apps when found</string>
-    <string name="prefs_updated_apps">Number of updated apps to show</string>
+    <string name="prefs_updated_apps">Number of non-new apps to show</string>
```

Comment:
Nah, it's too much not on point, it's specifically about the apps that got an update (sorted chrono.)

Reply:
Yes, I thought about it.